.travis.yml

```diff
@@ -3,16 +3,14 @@ sudo: required
 
 dart:
   - 1.24.3
-  - stable
+  - dev
 dart_task:
-  - test: --platform vm
-  - test: --platform dartium
-  - install_dartium: true
+  - dartfmt: true
+  - dartanalyzer: lib example tool test/unit
 
 script:
   - pub publish --dry-run # Validate package
   - pub run dart_dev gen-test-runner --check
-  - pub run dart_dev format --check
```

Comment:
At least right now, this takes configuration from dev.dart, so should we keep it?

Reply:
either way. I was experimenting with trying to get rid of the dart_style dependency .. but ended up restoring it anyway... so no reason why we can't restore this to dart_dev format.